release-scripts/build-conda-tgz.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Script used to install cmdstan on a GColab Machine and create a .tgz archive with the installation
+
+# Arguments:
+# $1: cmdstan version in the form of 2.29.0 (major.minor.patch)
+
+# On a fresh colab instance simply run:
+# !curl https://raw.githubusercontent.com/stan-dev/ci-scripts/conda-tgz-script/release-scripts/build-conda-tgz.sh | bash -s -- 2.29.0
```

Comment:
Before merging we should update this URL to what it will be (I think just delete `conda-tgz-script`?)

Reply:
I've changed it to master, if we do not specify a branch it leads to a 404.
Just tried with:
- `curl https://raw.githubusercontent.com/stan-dev/ci-scripts/master/release-scripts/functions.sh`
- `curl https://raw.githubusercontent.com/stan-dev/ci-scripts/release-scripts/functions.sh`